### ISSUE TITLE
argument of only underscores results in "Use of uninitialized value in ..." error

### ISSUE THREAD

**Issue by tipdbmp** [kablamo/perl-string-camelsnakekebab#2]
Calling the functions:

``` perl
use String::CamelSnakeKebab qw|:all|;
say lower_camel_case('_');
say upper_camel_case('__');
say lower_snake_case('___');
say upper_snake_case('____');
say constant_case('_____');
say kebab_case('______');
say http_header_case('________');
```

with an argument string which contains only underscores results in an `"Use of uninitialized value in ..."` error

**Comment by tsibley** [comment]
@kablamo Looks like this could be closed given 52b201f.

**Comment by kablamo** [comment]
Thanks!
